saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SemanticRoleLabeling/PopulateSRLDataModel.scala

```diff
@@ -44,24 +44,16 @@ object PopulateSRLDataModel extends Logging {
     val annotatorService = useCurator match {
       case true =>
         val nonDefaultProps = new Properties()
-        nonDefaultProps.setProperty(CuratorConfigurator.RESPECT_TOKENIZATION.key, Configurator.TRUE)
+        TextAnnotationFactory.enableSettings(nonDefaultProps, RESPECT_TOKENIZATION)
         CuratorFactory.buildCuratorClient(
           new CuratorConfigurator().getConfig(new ResourceManager(nonDefaultProps))
         )
       case false =>
         val nonDefaultProps = new Properties()
+        TextAnnotationFactory.disableSettings(nonDefaultProps, USE_NER_CONLL, USE_NER_ONTONOTES, USE_SRL_VERB, USE_SRL_NOM, USE_STANFORD_DEP)
```

Comment:
@danyaljj @kordjamshidi : I've changed this to reuse `TextAnnotationFactory`, please check if there is any problem.